### PR TITLE
In search for menu commands, replace 'Application' with 'Photoshop CC…

### PR DESCRIPTION
**Pull request by hamburge**
…' on Mac

For issue #1868

### CODE REVIEW

File: src/js/actions/search/menucommands.js [hunk 1]
```diff
@@ -53,7 +53,16 @@ define(function (require, exports) {
             }
 
             if (labels[part].$MENU) {
-                path += labels[part].$MENU + ">";
+                // Replace "Application" with "Photoshop CC" on Macs
+                // Don't show "Application" options on Windows
+                if (part !== "APPLICATION") {
+                    path += labels[part].$MENU + ">";
+                } else if (system.isMac) {
+                    path += "Photoshop CC >";
```
Comment (iwehrman) [inline src/js/actions/search/menucommands.js]: At the very least this needs to be localized. But, what happens if you just change this line? https://github.com/adobe-photoshop/spaces-design/blob/master/src/nls/root/strings.js#L68

**Comment by hamburge** [comment]
@iwehrman Yep, that did it. I can't test on Windows though...

File: src/js/actions/search/menucommands.js [hunk 2]
```diff
@@ -53,7 +53,13 @@ define(function (require, exports) {
             }
 
             if (labels[part].$MENU) {
-                path += labels[part].$MENU + ">";
+                // Don't show "Application" options on Windows
```
Comment (iwehrman) [inline src/js/actions/search/menucommands.js]: I don't think any of this is necessary? There is no "application" menu on Windows...

Comment (iwehrman) [inline src/js/actions/search/menucommands.js]: Which is to say that it seems like the one-line diff to the strings file is all that should be needed?

**Comment by hamburge** [comment]
@iwehrman Ok reverted it back to what we had before except for the strings file

**Comment by iwehrman** [comment]
:+1:

**Pull request merged**